editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -3931,9 +3932,18 @@ void Node3DEditorViewport::focus_selection() {
 	cursor.pos = center;
 }
 
-void Node3DEditorViewport::assign_pending_data_pointers(Node3D *p_preview_node, AABB *p_preview_bounds, AcceptDialog *p_accept) {
+void Node3DEditorViewport::assign_pending_data_pointers(
+		Node3D *p_preview_node,
+		AABB *p_preview_bounds,
+		Ref<Material> *p_preview_material,
```

Comment:
Weird, a pointer to a reference? (pointer?)

Reply:
Yeah, hacky. Will likely be rewritten

Reply:
Should be resolved now